TestON/tests/PingallExample/PingallExample.py

```diff
@@ -152,7 +152,19 @@ def CASE3( self, main ):
 
         # install onos-app-fwd
         main.log.info( "Install reactive forwarding app" )
-        main.ONOScli1.featureInstall( "onos-app-fwd" )
+        import json
+        main.log.warn(" pretty print json")
+        main.log.debug( json.dumps( json.loads( main.ONOScli1.apps() ),
+                                    sort_keys=True,
+                                    indent=4, separators=(',',':') ) )
+        main.log.warn(" get the status of the app")
```

Comment:
Just so it would be a different color. I will remove these messages before pushing to master anyways